IdentityCore/src/controllers/MSIDSilentController.m

```diff
@@ -118,33 +118,85 @@ - (void)acquireTokenWithRequest:(MSIDSilentTokenRequest *)request
         if (result || !self.fallbackController)
         {
 #if !EXCLUDE_FROM_MSALCPP
-            MSIDTelemetryAPIEvent *telemetryEvent = [self telemetryAPIEvent];
-            [telemetryEvent setUserInformation:result.account];
-            [telemetryEvent setIsExtendedLifeTimeToken:result.extendedLifeTimeToken ? MSID_TELEMETRY_VALUE_YES : MSID_TELEMETRY_VALUE_NO];
-            [self stopTelemetryEvent:telemetryEvent error:error];
+            [self addTelemetryEvent:result isSsoExtFallBack:YES error:error];
 #endif
             self.currentRequest = nil;
-            
             completionBlock(result, error);
             return;
         }
 
-        self.currentRequest = nil;
         MSIDRequestCompletionBlock completionBlockWrapper = ^(MSIDTokenResult *ssoResult, NSError *ssoError)
         {
             // We don't have any meaningful information from fallback controller (edge case of SSO error) so we use the local controller result earlier
-            if (!ssoResult && (ssoError.code == MSIDErrorSSOExtensionUnexpectedError))
-            {
-                completionBlock(result, error);
-            }
-            else
+            
+            // If ssoError presented already, skip broker when came back from local RT fallback
```

Comment:
Can we update/delete the old comment above?

Reply:
Updated, moved to correct place